index.html

```diff
@@ -228,14 +228,31 @@ <h1 class="contact-title">
                </p>
             </div>
 
-            <div class="contact-form">
-               <input type="text" placeholder="Text Input" class="user-text">
-               <input type="text" name="email" id="email" placeholder="kibirigejohn005@gmail.com" class="user-email">
-               <textarea name="text-area" id="user-text" cols="30" rows="10" class="user-text-area" placeholder="Enter text here"></textarea>
-               <button class="see-projects project-btn">
-                  See Project
-               </button>
-            </div>
+            <form class="contact-form" action="https://formspree.io/f/xnqwkjoy" method="post">
```

Comment:
This is how your design looks on my end (using Google Chrome).

<img width="225" alt="Screen Shot 2022-07-05 at 13 35 24" src="https://user-images.githubusercontent.com/82426682/177393277-3099e776-6eec-4b3a-833f-3d9b7e4ce6ef.png">

Please fix it.

<img width="899" alt="Screen Shot 2022-07-05 at 13 35 43" src="https://user-images.githubusercontent.com/82426682/177393324-6b3018c9-90e1-4132-9e74-4289a7943049.png">

Reply:
Hey @vechicin  !! Thanks for the comment, however, I hope you adjusted the screen size to 375px as the ideal mobile size.